src/modules/fancyzones/editor/FancyZonesEditor/Models/Settings.cs

```diff
@@ -181,6 +181,12 @@ public static float Dpi
         }
         private static float _dpi;
 
+        private int ReadRegistryInt(string valueName, int defaultValue)
+        {
+            object obj = Registry.GetValue(_uniqueRegistryPath, valueName, defaultValue);

```

Comment:
`Registry.GetValue` could throw, let's add a try/catch handler while we're adding that wrapper

Reply:
We should not hide a coding bug, that call should not throw because those keys should be there.
If we try catch we, we would be failing silently.
We will move to json anyway, so no need to try to improve a code that will go away.

Reply:
@enricogior It could throw if a user doesn't have a Security permission though, the method simply returns `defaultValue` if the requested value isn't there.

Reply:
It can throw `SecurityException`, `IOException` and `ArgumentException`. None of these exceptions actually represent scenario that keys are not there. If the key is not there, on specified path, `Registry.GetValue` will return `null` according to documentation, and that is why this handler is here. If the keys are not there, which is perfectly normal thing to happen from my perspective, this handler will replace them with provided default value and continue.

Reply:
If a user doesn't have access to the registry, nothing will work, so no point in catching it anyway.